apps/web/src/stores/evaluations.ts

```diff
@@ -20,27 +21,36 @@ export default function useEvaluations(
   const { onSuccessCreate } = opts
   const { toast } = useToast()
 
+  const { documentUuid } = opts?.params ?? {}
+
   const {
     data = [],
     mutate,
     isLoading,
     error: swrError,
   } = useSWR<EvaluationDto[]>(
-    compact(['evaluations', ...flatten(Object.entries(opts?.params ?? {}))]),
-    async () => {
-      const [data, error] = await fetchEvaluationsAction(opts?.params)
+    compact(['evaluations', documentUuid]),
+    useCallback(async () => {
+      const route = documentUuid
+        ? _API_ROUTES.evaluations.detail({ documentUuid })
+        : _API_ROUTES.evaluations
+
+      const response = await fetch(route.root, { credentials: 'include' })
+
+      if (!response.ok) {
+        const error = await response.json()
 
-      if (error) {
         toast({
           title: 'Error fetching evaluations',
           description: error.formErrors?.[0] || error.message,
           variant: 'destructive',
         })
+
         throw error
```

Comment:
this is probably wrong we should not throw but instead return an empty array also you are throwing a json here